api/signup.go

```diff
@@ -88,6 +88,9 @@ func (a *API) Signup(w http.ResponseWriter, r *http.Request) error {
 			mailer := a.Mailer(ctx)
 			referrer := a.getReferrer(r)
 			if terr = sendConfirmation(tx, user, mailer, config.SMTP.MaxFrequency, referrer); terr != nil {
+				if errors.Is(terr, MaxFrequencyLimitError) {
+					return tooManyRequestsError("For security purposes, you can only request this once every minute")
```

Comment:
This looks better, but just one thing to note, gotrue actually has an env variable called `SMTP_MAX_FREQUENCY` which is used to rate limit the confirmation emails being sent out. I think it would be better to use that value in the error message to make it dynamic.

Reply:
Hey, just updated it. Sorry for the delay.